superwall/src/main/java/com/superwall/sdk/models/entitlements/RedeemRequest.kt

```diff
@@ -0,0 +1,25 @@
+package com.superwall.sdk.models.entitlements
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class RedeemRequest(
+    @SerialName("deviceId")
+    val deviceId: String,
+    @SerialName("appUserId")
+    val userId: String,
+    @SerialName("codes")
+    val codes: List<Reedemable>,
+)
+
+@Serializable
+data class Reedemable(
+    @SerialName("code")
+    val code: String,
```

Comment:
In the latest changes to the schema, there is now an optional `firstRedemption` flag that is either true or omitted. It's used to inform the backend that this is the first time the SDK has tried to redeem this code. This way we can know if we should automatically send an email on an expired code.